apps/filebrowser/src/filebrowser/utils.py

```diff
@@ -168,3 +181,15 @@ def is_file_upload_allowed(file_name):
       return False, f'File type "{file_type}" is restricted. Update file extension restrictions to allow this type.'
 
   return True, None
+
+
+def massage_stats(stats):
```

Comment:
Same here, please add an explanation of what this function does.

Reply:
Done